modules/campaignion_donation_type/campaignion_donation_type.features.uuid_node.inc

```diff
@@ -12,7 +12,7 @@ function campaignion_donation_type_uuid_features_default_content() {
   $nodes = array();
 
   $nodes[] = array(
-  'title' => 'Default Donation Form',
+  'title' => 'Default Donation Form - DO NOT DELETE',
```

Comment:
Maybe also changing it to „Donation form **template** - DO NOT DELETE” would be still clearer.

Reply:
I woul also remove the „Default …” in the beginning then.

Reply:
Was there a discussion about changing the node titles anywhere? I think this is something we should get @dayjess and/or @clouised28 involved in.

Reply:
Yes, the discussion was started by Jess. See link to mattermost conversation in ticket.

Reply:
I would also leave the "Default" in the title, for the reason that there can be many templates, but this is the original, "default" one.